src/main.js

```diff
@@ -184,6 +173,9 @@ function makePosterForm() {
 function showSavedPosters() {
   mainPoster.classList.add('hidden');
   savedPostersPg.classList.remove('hidden');
+  savedPostersGrid.innerHTML {
```

Comment:
If you're trying to change the savedPostersGrid here, the syntax is `savedPostersGrid.innerHTML = <newHTML>`. You can also use `appendChild` if you want to add things to the end of this HTML element: https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild.

Line 176 is currently causing an error due to these extra curly braces.